sbnanaobj/StandardRecord/SRBlip.h

```diff
@@ -0,0 +1,44 @@
+////////////////////////////////////////////////////////////////////////
+// \file    SRBlip.h
+// \brief   SRBlip object for localized energy deposits in bulk LAr
+// \author  $Author: jmclaughlin2@illinoistech.edu
+////////////////////////////////////////////////////////////////////////
+#ifndef SRBLIP_H
+#define SRBLIP_H
+#include "sbnanaobj/StandardRecord/SRVector3D.h"
+#include "sbnanaobj/StandardRecord/SRBlipHitClust.h"
+#include "sbnanaobj/StandardRecord/SRBlipTrueBlip.h"
+#include "sbnanaobj/StandardRecord/SRCVNScore.h"
+
+namespace caf
+{
+  const int kNplanes    = 3;
+    class SRBlip
+    {
+    public:
+        SRBlip();
+        virtual ~SRBlip();
+        int       ID;         // Blip ID / index
+        bool      isValid;      // Blip passes basic checks
+        int       Cryostat;         // Cryostat
+        int       TPC;         // TPC
+        int       NPlanes;         // Num. matched planes
+        int       MaxWireSpan;         // Maximum span of wires on any plane cluster
+        float     TimeTick;       // Readout time [ticks]
+        float     Time;       // Drift time [us]
+        float     Charge;         // Charge on calorimetry plane
+        float     Energy;       // Energy (const dE/dx, fcl-configurable) [GeV]
+        float     EnergyESTAR;       // Energy (ESTAR method from ArgoNeuT) [GeV]
+        float     EnergyPSTAR;       // Energy (PSTAR method similar with ESTAR method from ArgoNeuT) [GeV]
+        float     ProxTrkDist;         // Distance to cloest track
+        int       ProxTrkID;         // ID of closest track
```

Comment:
Is this an ID that can be found on a track object of `rec.slc.reco.pfp.id`? (in that case, it's not a _track_ ID but a PFP ID) Or is it a GEANT4 track, and that ID can be found in the truth information?